src/common/constants.ts

```diff
@@ -117,14 +117,18 @@ export abstract class SkyboxImages {
 export abstract class PlayerConstants {
   static readonly playerSize: [number, number, number] = [1, 1, 0.125];
 
-  static readonly playerSpawnPosition: [number, number, number] = [0, 3, 0];
-
   static readonly movementSpeed = 5;
 
   static readonly jumpSpeed = 6;
 
   static readonly playerHeight = 2;
 
+  static readonly playerSpawnPosition: [number, number, number] = [
```

Comment:
```suggestion
  static readonly height = 2;

  static readonly spawnPosition: [number, number, number] = [
```

more clean imo to do PlayerConstants.spawnPosition -- no need for player twice. applies everywhere

Reply:
Good point. I can change this universally